src/extension.js

```diff
@@ -410,17 +319,55 @@ const ServiceIndicator = GObject.registerClass({
         this.settings.run_dispose();
 
         // Destroy the PanelMenu.SystemIndicator actors
-        this._item.destroy();
         this.menu.destroy();
 
-        delete AggregateMenu._gsconnect;
+        delete QuickSettingsMenu._gsconnect;
```

Comment:
This line can probably just be removed, it was a weird old compatibility thing with other extensions.